docs/en_US/Compressor/Pruner.md

```diff
@@ -92,3 +92,55 @@ You can view example for more information
 
 ***
 
+## Filter Pruner
+
+In ['PRUNING FILTERS FOR EFFICIENT CONVNETS'](https://arxiv.org/abs/1608.08710), authors Hao Li, Asim Kadav, Igor Durdanovic, Hanan Samet and Hans Peter Graf.
+
+![](../../img/filter_pruner.png)
+
+> The procedure of pruning m filters from the ith convolutional layer is as follows:
+> 1. For each filter $F_{i,j}$ , calculate the sum of its absolute kernel weights $s_j = \sum_{l=1}^{n_i}\sum|K_l|$
```

Comment:
what is the meaning of K?

Reply:
K is a convolution kernel, often 3x3 matrix